python/lsst/ts/xml/data/sal_interfaces/GIS/GIS_Events.xml

```diff
@@ -31,4 +31,1674 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>GIS</Subsystem>
+    <EFDB_Topic>GIS_logevent_gisCpuInputs</EFDB_Topic>
+    <item>
+      <EFDB_Name>sdiCPUetw1A</EFDB_Name>
+      <Description>Wireless Emergency pushbutton1 channel1.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiCPUetw1B</EFDB_Name>
+      <Description>Wireless Emergency pushbutton 1 channel2.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiCPUetw2A</EFDB_Name>
+      <Description>Wireless Emergency pushbutton 2 channel1.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiCPUetw2B</EFDB_Name>
+      <Description>Wireless Emergency pushbutton 2 channel2.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>sdiCPUfree1</EFDB_Name>
+      <Description>Reserve 1.</Description>
```

Comment:
Please change to `Reserved for future use XXX` with XXX the number that's now in the description here and everywhere else.